pina/solvers/pinn.py

```diff
@@ -97,6 +97,15 @@ def configure_optimizers(self):
         """
         return self.optimizers, [self.scheduler]
     
+    def _loss_data(self, input, output):
```

Comment:
also `_loss_data` and `_loss_phys` I would make it public. In such a way if the user wants to define a new PINN (e.g. gradient-PINN, weak-PINN, ...) he needs just to overwrite those functions.

Reply:
Private and public methods can be redefined in child classes!
Private methods/attributes just identify what shouldn't be called from external classes